common/src/main/scala/com/gu/media/youtube/YouTubeClaims.scala

```diff
@@ -0,0 +1,126 @@
+package com.gu.media.youtube
+
+import com.google.api.services.youtubePartner.YouTubePartner
+import com.google.api.services.youtubePartner.model._
+import com.gu.media.logging.Logging
+import com.typesafe.config.Config
+import org.joda.time.DateTime
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+
+class YouTubeClaims(override val config: Config) extends YouTubeAccess with Logging {
+
+  private def createAsset(title: String, videoId: String): String = {
```

Comment:
Yes, this is confusing, it should return `Asset` and then get the id string from that.